Enhance trace_view.md with span focus feature (#37544)
DataDog/documentation: content/en/tracing/trace_explorer/trace_view.md

{{% /tab %}}
{{< /tabs >}}

### Focus on a span

> In the Waterfall and Flamegraph visualizations, you can focus on a span to narrow the view to that span and its children. This is useful for large traces, where focusing lets you isolate and investigate a specific group of operations without the rest of the trace adding noise.

Reviewer: ```suggestion
In the waterfall and flame graph visualizations, you can focus on a span to narrow the view to that span and its children. This is useful for large traces, where focusing lets you isolate and investigate a specific group of operations without the rest of the trace adding noise.
```

> In the Flamegraph, select a span, then click the focus icon that appears next to the minimap. In the Waterfall, hover over a span and click the focus icon that appears on its right side. The visualization rescales to the selected span as the new root and its descendants fill the timeline. To return to the full trace, reset the focus.

Reviewer: ```suggestion

In the flame graph, select a span, then click the focus icon next to the minimap. In the waterfall, hover over a span and click the focus icon on its right side. The visualization rescales with the selected span as the new root, and its descendants fill the timeline. To return to the full trace, reset the focus.
```

Reviewer: Adding an extra line so that this renders as a separate paragraph on the page.

{{< img src="/tracing/trace_view/flamegraph-focus-on-span.mp4" alt="Flamegraph focus on span" video="true" style="width:90%;">}}
{{< img src="tracing/trace_view/waterfall-focus-on-spans.mp4" alt="Waterfall focus on span" video="true" style="width:90%;">}}

> Note: Focusing only changes what is displayed. It does not filter spans out of search results or alter the underlying trace.

Reviewer: ```suggestion
<div class="alert alert-info">Focusing only changes what is displayed. It does not filter spans out of search results or alter the underlying trace.</div>
```
